File: src/main/java/net/atos/zac/policy/input/TaakData.java
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.policy.input;

import jakarta.json.bind.annotation.JsonbProperty;

public class TaakData {

    public boolean open;

    @JsonbProperty("zaak_open")
    public boolean zaakOpen;

    public String zaaktype;

}

File: src/main/kotlin/nl/info/zac/policy/input/TaakData.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2025 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.policy.input

import jakarta.json.bind.annotation.JsonbProperty

data class TaakData(
    @field:JsonbProperty("open")
    val open: Boolean = false,

    @field:JsonbProperty("zaaktype")
    val zaaktype: String? = null
)
